reliability-engineering/terraform/modules/concourse-web/launch-template.tf

```diff
@@ -10,6 +10,15 @@ data "aws_ami" "ubuntu_bionic" {
   }
 }
 
+data "null_data_source" "local_users" {
+  count = "${var.worker_team_names}"
+
+  user = "${element(var.worker_team_names, count.index)}-bot:${lookup(
```

Comment:
Should this be:

```
inputs = {
  user = "${element(var.worker_team_names, count.index)}-bot:${lookup(
    var.local_user_passwords,
    element(var.worker_team_names, count.index)
  )}"
}
```

Reply:
Mind blown. Will change.

Reply:
I think this could actually be

```
locals {
  usernames_and_passwords = "${join(",", formatlist(
    "%s:%s",
    keys(map_of_usernames/passwords),
    values(map_of_usernames/passwords),
  ))}"
}
```